spring-pulsar-spring-cloud-stream-binder/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'org.springframework.pulsar.spring-module'
+}
+
+description = 'Spring Cloud Stream Binder for Apache Pulsar'
+
+dependencies {
+    api project (':spring-pulsar')
+    api('org.springframework.cloud:spring-cloud-stream') {
+        exclude group: 'javax.activation', module: 'javax.activation-api'
```

Comment:
How did you know to exclude these? Is there any other baggage we want to exclude?

Reply:
Build complained.